Node/src/TeamsChatConnector.ts

```diff
@@ -167,21 +175,29 @@ export class TeamsChatConnector extends builder.ChatConnector {
       // Filter out @mention
       var out: builder.IEvent[] = []; 
       for (var re of realEvents) {
-        let intentMessage = <IntentMessage>re;
+        let intentMessage = <builder.IMessage>re;
         // Add intent message
-        if (intentMessage.type == 'message') {
-          var atMentions = [];
-          intentMessage.intentText = intentMessage.text;
+        if (this.usingIntentText && intentMessage.type == 'message') {
+          let atMentions = [];
```

Comment:
Still not clear why we need this. (Plus it'd be "botAtMentions", since this isn't all mentions, just the bot mentions).